server/.eslintrc

```diff
@@ -5,7 +5,8 @@
     "jest/globals": true
   },
   "rules": {
-    "no-underscore-dangle": ["error", { "allowAfterThis": true }]
+    "no-underscore-dangle": ["error", { "allowAfterThis": true }],
+    "import/no-extraneous-dependencies": ["error", {"devDependencies": true}]
```

Comment:
I added this because it kept complaining about extraneous dev-dependencies and I wanted to commit my work. This might not be necessary/helpful.